docs/csharp/misc/cs0272.md

```diff
@@ -10,14 +10,16 @@ ms.assetid: 16a9aab6-922a-45a3-a0ef-f32e99f3950f
 ---
 # Compiler Error CS0272
 
-The property or indexer 'property/indexer' cannot be used in this context because the set accessor is inaccessible  
-  
- This error occurs when the `set` accessor is not accessible to the program code. To resolve this error, increase the accessibility of the accessor, or change the calling location. For more information, see [Restricting Accessor Accessibility](../programming-guide/classes-and-structs/restricting-accessor-accessibility.md).  
-  
-## Example  
+The property or indexer 'property/indexer' cannot be used in this context because the set accessor is inaccessible, this error occurs when the `set` accessor is not accessible to the program code.
```

Comment:
Let's keep these as separate paragraphs. The first sentence is the error message produced by the compiler. Having it stand alone increases SEO for that text:

```suggestion
The property or indexer 'property/indexer' cannot be used in this context because the set accessor is inaccessible.

 This error occurs when the `set` accessor is not accessible to the program code.
```